CHANGELOG.md

```diff
@@ -13,6 +13,9 @@ About changelog [here](https://keepachangelog.com/en/1.0.0/)
 - Search for cases phenotypically similiar to a case also from WUI.
 - Case variant searches can be limited to similar cases, matching HPO-terms,
   phenogroups and cohorts.
+- Dearchive reruns and flad them as 'inactive' if archived
+- Sort cases by analysis_date, track or status
+- Display cases in the following order: prioritized, active, inactive, archived, solved
```

Comment:
From a user perspective, the order should be prio, inactive, active, archived, solved. Possibly, one could separate active-assigned from active-unassigned as well - the prior may be mistaken opens that should really be inactive. But, it is possible to search for them.

Reply:
Actually, it would not be bad if a cases is auto-assigned to the person activating it - and returned to inactive status (but with audit log intact naturally) when the last assigned user un-assigns from it.

Reply:
> From a user perspective, the order should be prio, inactive, active, archived, solved.

That's why asked for your review, thanks!

Reply:
> Actually, it would not be bad if a case is auto-assigned to the person activating it - and returned to inactive status (but with audit log intact naturally) when the last assigned user un-assigns from it.

Good idea I'll add to this PR!

Reply:
Done! Take a look when you have time, it should work. Too bad I still can't login to clinical-db to test it on stage